From 0aafe11bc33cef5973ee97baeca64e98481a25ce Mon Sep 17 00:00:00 2001
From: Greg Kroah-Hartman <gregkh@suse.de>
Date: Thu, 4 Dec 2008 20:19:44 -0800
Subject: Staging: rtl8187se: make the built module be the proper name

From: Greg Kroah-Hartman <gregkh@suse.de>

If we are calling this the rtl8187se module, let's actually build
a module with that name.

Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/rtl8187se/Makefile |    5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

--- a/drivers/staging/rtl8187se/Makefile
+++ b/drivers/staging/rtl8187se/Makefile
@@ -26,7 +26,8 @@ EXTRA_CFLAGS += -DENABLE_LPS
 
 #EXTRA_CFLAGS += -mhard-float -DCONFIG_FORCE_HARD_FLOAT=y
 
-r8187se-objs :=	r8180_core.o		\
+rtl8187se-objs :=			\
+		r8180_core.o		\
 		r8180_sa2400.o		\
 		r8180_93cx6.o		\
 		r8180_wx.o		\
@@ -50,5 +51,5 @@ r8187se-objs :=	r8180_core.o		\
 		ieee80211/ieee80211_crypt_ccmp.o	\
 		ieee80211/ieee80211_crypt_wep.o
 
-obj-$(CONFIG_RTL8187SE)	+= r8187se.o
+obj-$(CONFIG_RTL8187SE)	+= rtl8187se.o
 
